### DISCUSSION TITLE
网络请求

### DISCUSSION THREAD

**Discussion by MUYI-XIAN** [simple-robot/Discussions#71] [酒馆]
### 选择此问题涉及的编程语言

Java

### 疑问描述

请问如何通过网络请求获取响应的json文件

**Comment by ForteScarlet** [comment]
？

**Reply by MUYI-XIAN** [reply]
前端有axios这种比较方便的工具，但是原生java得自己写统一处理的工具，想知道simbot有没有那种写好工具类。我翻阅了文档看到了类似的包（simbot-component-http-server-api），但是idea并没有它下面的类，请问是我的依赖没导入吗？

**Reply by ForteScarlet** [reply]
Java网络相关的库一大把，自行通过搜索引擎搜索，simbot没有提供其他封装。
`simbot-component-http-server-api` 是尚在计划中的api组件，与http工具无关。
